backend/app/db/database_service.py

```diff
@@ -75,54 +77,59 @@ def format_genres() -> None:
         crud.update_genre_name(db, genre)
 
 
-def init_meilisearch_indexing():
+def init_meilisearch_indexing(chunk_size: int):
     """MeiliSearch indexing from Postgres DB
     """
-    supported_country_codes = get_settings().supported_country_codes
 
-    for country_code in supported_country_codes:
-        db = database.SessionLocal()
-        all_media_iter = crud.get_all_media_iter(db=db)
-        db.close()
-
-        client.index(f'media_{country_code}').add_documents([
-            schemas.Media(
-                id=media.id,
-                title=media.title,
-                original_title=media.original_title,
-                overview=media.overview,
-                release_date=media.release_date,
-                genres=media.genres,
-                poster_path=media.poster_path,
-                popularity=media.popularity,
-                specific_provider_names=[
-                    provider.get(country_code).get('provider_name')
-                    for provider in media.providers
-                    if provider.get(country_code)
-                ],
-                specific_providers=[
-                    provider.get(country_code)
-                    for provider in media.providers
-                    if provider.get(country_code)
-                ]
-            ).dict()
-            for media in all_media_iter
-        ])
-
-        extract_unique_providers_to_txt(all_media_iter, country_code)
-
-
-def extract_unique_providers_to_txt(media_list, country_code):
-    provider_set = {
-        provider.get(country_code).get('provider_name')
-        for media in media_list
-        for provider in media.providers
-        if provider.get(country_code)
-    }
-    ordered_provider_list = sorted(provider_set)
-    with open(f'../providers_{country_code}.txt', 'w') as file:
-        for provider in ordered_provider_list:
-            file.write(f'{provider}\n')
+    for country_code in get_settings().supported_country_codes:
```

Comment:
Should we use `tqdm` around this loop instead of the `print(f"Queuing up {country_code} media for MeiliSearch indexing")`? This will make a lot of noise when we support many countries.

Reply:
Got the pbar now, but I refactored it a bit, see [comment below](https://github.com/streamchaser/streamchaser/pull/118#issuecomment-1018642626)